charts/tf-controller/templates/runner-discovery-service.yaml

```diff
@@ -0,0 +1,16 @@
+{{- range include "tf-controller.runner.allowedNamespaces" . | fromJsonArray }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: tf-runner
+  namespace: {{ . }}
+spec:
+  clusterIP: None
+  ports:
+  - name: grpc
+    port: 30000
+  selector:
+    app.kubernetes.io/created-by: tf-controller
+		app.kubernetes.io/name: tf-runner
+{{- end }}
```

Comment:
Hi @syalioune 

We want to rely only on Pod hostnames without introducing the Service layer.
Would it be possible to do so with your hacks?

Reply:
A service is mandatory in my proposal to have a consistent DNS entry generated regardless of the k8s cluster setup.
A service with pod hostnames/subdomains is the best compromise since it requires only one service per namespace instead of a service per runner pod.